### ISSUE TITLE
On viewgame page, lack of alt text on small cover art can be an issue for screen reader users

### ISSUE THREAD

**Issue by alice-blue** [iftechfoundation/ifdb#1378]
[Webaim.org](https://webaim.org/techniques/alttext/#functional) says “When an image is the only content inside a link or button, alt text is all that a screen reader has to go on. If it’s empty or missing, screen readers might read the image filename or the URL of the linked page in hopes that this may be helpful to the user, but this is not always the case.”

An IFDB user [says](https://intfiction.org/t/poll-alt-text-for-cover-art-on-ifdb/80337/11):

> Yeah, that’s true in my experience. I think that’s what happens for the main cover image on the ifdb page. For Lost Pig, for example, immediately before the level 1 heading I get this: viewgame coverart&id=mohwfk47yâ¦"
Which is presumably the link to the page with more image info.

Text would be an improvement over reading aloud a URL, and [voters are in favor of identifying the image and telling what the link leads to](https://intfiction.org/t/poll-alt-text-for-cover-art-on-ifdb/80337), so we should put alt text there instead of leaving the alt text blank.
